crates/noirc_driver/src/main.rs

```diff
@@ -14,18 +11,19 @@ fn main() {
     let mut context = Context::default();
 
     // Add local crate to dep graph
-    create_local_crate(&mut context, ROOT_DIR_MAIN, CrateType::Binary);
+    let local_crate_id = create_local_crate(&mut context, ROOT_DIR_MAIN, CrateType::Binary);
```

Comment:
Should we store the root crate id in the context instead? When would we pass a different id instead when compiling?

Reply:
No. The we should be working with whatever crate_id is created in the graph and it should not be stored. This moves us towards being able to compile any node in the graph.